zkstack_cli/crates/zkstack/src/commands/chain/args/init/mod.rs

```diff
@@ -80,14 +92,40 @@ impl InitArgs {
             })
         };
 
-        InitArgsFinal {
+        let ecosystem_contracts_path = if self.dev {
+            self.ecosystem_contracts_path.map_or_else(
+                || {
+                    ecosystem.as_ref().map_or_else(
+                        || chain.get_preexisting_ecosystem_contracts_path(),
+                        |e| e.get_contracts_path(),
+                    )
+                },
+                PathBuf::from,
+            )
+        } else if let Some(ecosystem) = &ecosystem {
+            ecosystem.get_contracts_path()
+        } else {
+            get_ecosystem_contracts_path(self.ecosystem_contracts_path, ecosystem.clone(), chain)?
+        };
+
+        let wallets_path = ecosystem.map_or_else(
+            || {
+                self.wallets_path
+                    .map_or_else(|| Prompt::new(MSG_WALLETS_PATH_PROMPT).ask(), PathBuf::from)
+            },
+            |e| e.get_wallets_path(),
+        );
+
```

Comment:
from messages it's not clear what contracts and wallets you want

Reply:
I updated the wallets message in: [f194e18](https://github.com/matter-labs/zksync-era/pull/3231/commits/f194e18734715dd47ccbe149c70611a53e6afa71)

Is this OK or what should the message be?

Reply:
Prompt for contracts path is:
`Provide the path to the ecosystem contracts or keep it empty and you will use ZKsync ecosystem config. For using this config, you need to have governance wallet`
How can we improve it?